README.md

```diff
@@ -95,6 +95,14 @@ tach clean
 This will wipe all the configuration generated and enforced by Tach.
 
 
+If you just want to see the external dependencies and external usage of a given file or module in your project, you can use `tach report`.
+
+```bash
+tach report my_package
```

Comment:
```suggestion
tach report my_package/
```
Just to clearly differentiate from the other use case

Reply:
Wonder if we'd want to support list of items here similar to `--exclude` in the future as well